Apps/Src/CommandLine.c

```diff
@@ -30,7 +30,7 @@ static bool cmd_Contactors_Set(void);
 
 static bool cmd_Minion_Read_Input(void);
 
-static bool cmd_Minion_Write_Output(void);
+static bool cmd_Minion_write(void);
```

Comment:
Why lowercase on the write/read? All of our other drivers are uppercase after the driver name. We should be consistent

Reply:
I think it makes sense for functions to be camelCase with the first letter being undercase. But if it is difference in other drivers then I'll make the change